compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -105,6 +105,13 @@ enum class VectorPreProcStrategy {
   None
 };
 
+static llvm::cl::opt<bool> clArmSMETiling(
+    "iree-llvmcpu-arm-sme-tiling",
+    llvm::cl::desc("When targetting AArch64 with SME enabled (+sme), use this "
+                   "option to disable SME tiling (enabled by default). This "
+                   "will make IREE effectively target SSVE rather than SME."),
```

Comment:
> When targeting AArch64 with SME enabled (+sme), use this option to disable SME tiling (enabled by default). This will make IREE effectively target SSVE rather than SME.

nit: I don't think this quite describes the option (it does not disable anything, when its on it _enables_ the tiling). 

Maybe something like:
> Flag to control if SME tiling should be used on AArch64 with SME enabled (+sme). This defaults to true, but can be disabled by explicitly setting this flag to false.

Reply:
Also:
> This will make IREE effectively target SSVE rather than SME.

(Not necessarily true, as that's enabled by another mechanism)